pkg/service/schema.go

```diff
@@ -111,6 +173,48 @@ func (i IntPropertySchema) MarshalJSON() ([]byte, error) {
 	)
 }
 
+func (i IntPropertySchema) validate(context string, value interface{}) error {
+	floatVal, ok := value.(float64)
+	if !ok {
+		return NewValidationError(context, "field value is not of type int64")
```

Comment:
should we return int64 as part of the validation error? JSON Schema really only defines "integer" and "number". Might be better to return the generic type integer here.

Reply:
👍